### PR TITLE
[fix](olap) remove zorder support when create table

### PR DESCRIPTION
**Pull request by nextdreamblue**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by nextdreamblue** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/common/util/PropertyAnalyzer.java [hunk 1]
```diff
@@ -17,6 +17,12 @@
 
 package org.apache.doris.common.util;
 
+import com.google.common.collect.Maps;
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/common/util/PropertyAnalyzer.java:20]: Import order is wrong

Comment (nextdreamblue) [reply fe/fe-core/src/main/java/org/apache/doris/common/util/PropertyAnalyzer.java]: done

**Comment by nextdreamblue** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Comment by ptallada** [comment]
Hi,

I cannot find any information about why this feature was removed.
Is there any link to any discussion about the reasons?

Thanks!

**Pull request merged**